src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_commands.py

```diff
@@ -47,6 +47,77 @@ def test_containerapp_env_e2e(self, resource_group):
             JMESPathCheck('length(@)', 0),
         ])
 
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="australiaeast")
+    @StorageAccountPreparer(location="australiaeast")
+    @live_only()  # encounters 'CannotOverwriteExistingCassetteException' only when run from recording (passes when run live)
```

Comment:
May I ask what is the reason for this problem `'CannotOverwriteExistingCassetteException' only when run from recording`?

Reply:
If I recall correctly, this test actually passes locally (when run from a recording or live), but fails when run in the CI. Let me try rerecording again to see if it works

Reply:
Yes, I just double-checked this with a rerecorded test. The test passes locally, but fails the CI for some reason. I think the best thing to do here is mark it as live_only and investigate further when we have time

Reply:
Ok, please remember to submit PR separately to solve the `@live_only()` problem in the future. It is recommended that you create a work item for these testing problems